pytorch_lightning/plugins/training_type/training_type_plugin.py

```diff
@@ -57,6 +57,12 @@ def setup_environment(self) -> None:
     def setup(self, model: Module) -> None:
         """Called by the accelerator to finish setup."""
 
+    def reconciliate_processes(self, trace: str):
```

Comment:
we should consider adding this only to the parallel plugin and not to the base class.
what do you think @justusschock ?

Reply:
@awaelchli I thought about that as well, but then it wouldn't make sense to call this in the trainer unconditonally. Maybe we should rename this to `reconcilate_training`?

Reply:
Agree. Done !